Source/Core/Other/FoodUtility.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using PrisonLabor.Core.Needs;
+using PrisonLabor.Core.Trackers;
+using RimWorld;
+using Verse;
+using Verse.AI;
+
+namespace PrisonLabor.Core.Other
+{
+    public static class PrisonFoodUtility
```

Comment:
And what was wrong with old way to deliver food for prisoners?

Reply:
The problem was prisoners outside the prison cells won't get fed.

Reply:
Patch in `DeliverEvenOutsidePrisonCell` should handle that

Reply:
It works now.

Reply:
But you know that delivering food is wardens job and it should stay that way

Reply:
I'm leaving this as is until i redo the labor thinknode since that is way more of a problem to the performance.

Reply:
you're right but until the foundation is more optimized i rather have as few as possible harmony patches for the thinknodes